README.md

```diff
@@ -43,10 +43,10 @@ If you want a simple guide to install Empirical Core, then you've come to the ri
 
 9. Run Redis with ```redis-server```
 
-11. Run npm installer with ```npm install && cd ./client && npm install```
```

Comment:
I don't see those in the README.md in this PR. Are you referring to the main repo's README? Can you point to the lines in the README.md located in this PR?